developer-workflow/psrt.rst

```diff
@@ -0,0 +1,148 @@
+Python Security Response Team (PSRT)
+====================================
+
+The Python Security Response Team (PSRT) is responsible for handling
+vulnerability reports for CPython and pip.
+
+Vulnerability report triage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
---------------------------
```

Reply:
Resolved in https://github.com/python/devguide/pull/1348/commits/0344b7ce0b59c9ecebb41e28f353ed67ab704b30